Bangazon/Controllers/CustomersController.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+using Microsoft.AspNetCore.Http;
+using Bangazon.Models;
+
+//Aaron Keen
+//This controller allows you to get all customers, get a single customer, query the products the customer is selling, query the payment types the customer has used to pay, post a new customer, and edit an existing customer
+
+namespace Bangazon.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomersController : Controller
+    {
+        private readonly IConfiguration _config;
+
+        public CustomersController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public IDbConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+        //GET api/Customers?q=Taco
+        [HttpGet]
+        public async Task<IActionResult> Get(string q, string _include)
+        {
+            string sql = @"
+            SELECT
+                c.Id,
+                c.FirstName,
+                c.LastName
+            FROM Customer c
+            WHERE 1=1
+            ";
+
+            if (q != null)
+            {
+                string isQ = $@"
+                    AND c.FirstName LIKE '%{q}%'
+                    OR c.LastName LIKE '%{q}%'
+                ";
+                sql = $"{sql} {isQ}";
+            }
+
+            if (_include == "products")
```

Comment:
When _include=products is specified and a customer has not created any products, that customer is not being returned. I would expect to see the customer with an empty list of products.